test/e2e/values.yaml

```diff
@@ -0,0 +1,7 @@
+mapServices:
+  fromVirtual:
+  - from: test/test
+    to: test
+  fromHost:
+  - from: test/test
+    to: default/test
```

Comment:
Is this actually tested by any of the tests in the "e2e" suite?
If not, it still makes sense to add it, but perhaps to the `test/commonValues.yaml` instead. By adding it we would test that syncer doesn't crash when one of the mapped services is missing.

Reply:
It is tested in the e2e test suite. It's only specific to that test suit.

Reply:
Oh, I didn't know. Thanks for confirming. :+1: